compose/config/config.py

```diff
@@ -12,18 +12,13 @@
 from cached_property import cached_property
 
 from . import types
-from .. import const
 from ..const import COMPOSEFILE_V1 as V1
-from ..const import COMPOSEFILE_V2_1 as V2_1
-from ..const import COMPOSEFILE_V2_3 as V2_3
-from ..const import COMPOSEFILE_V3_0 as V3_0
-from ..const import COMPOSEFILE_V3_4 as V3_4
+from ..const import COMPOSEFILE_V4 as VERSION
```

Comment:
There's no such thing as a V4 composefile schema. Better use `COMPOSE_SPEC` here

Reply:
After a discussion with @ulyssessouza, we decided it was easier to name the merged schema v4. So far the compose_spec schema is the v3.9. I would find it confusing to name it as you suggested as long as we don't have these schemas aligned.

Reply:
then better to start by getting schema aligned :P
I'm fine with any name you want to use here, but please don't use "V4". This is highly confusing as this suggest a compatibility break between v3 and compose-spec, that we definitively want to avoid. There's no such thing like a v4 schema

Reply:
By the way, compose-spec schema to be 3.9 is obsolete now we decided to drop version field
=> https://github.com/compose-spec/compose-spec/pull/88